.coveragerc

```diff
@@ -1,4 +1,13 @@
 [run]
+    source=elephants_cse5911
+
+[report]
```

Comment:
Do we need these ignore statements? Which of our files are ignored by these?

Reply:
Codecov suggests having the source part, but the rest I will remove. 
https://github.com/codecov/example-python#how-to-generate-coverage-reports